src/sass/common.scss

```diff
@@ -331,15 +331,6 @@ fieldset .error-msg {
     position: absolute;
     top: -9999px;
 }
-.indicate-loading {
```

Comment:
Yeah, I did notice the several lines in bet_form.js and also in pjax.js but decided to leave them there if we decide to actually add some styling in the future. No harm.